requests/sessions.py

```diff
@@ -735,6 +735,9 @@ def session():
     """
     Returns a :class:`Session` for context-management.
 
+    This method is kept for backwards compatibility. For new code,
```

Comment:
Let's do something different since we're using sphinx + reStructuredText:

```rst
.. deprecated:: 1.0.0

    This method has been deprecated since version 1.0.0 and is only kept for
    backwards compatibility. New could must use :class:`~requests.session.Session`
    to create a session. This will be removed at a future date.
```